locales/en/common.json

```diff
@@ -1,36 +1,100 @@
+  "full_pagination": "Full pagination",
   "go_to_kafka_instances": "Go to $t(common:kafka) Instances",
   "id": "ID",
+  "input_filed_invalid_helper_text": "Must start with a letter and end with a letter or number. Valid characters include lowercase letters from a to z, numbers from 0 to 9, and hyphens ( - ).",
+  "input_text_area_invalid_helper_text": "Valid characters include letters from Aa to Zz, numbers from 0 to 9, spaces, and line breaks.",
+  "instance_checking_message": "Checking if new Service Registry instances are available",
+  "instance_information": "Instance information",
+  "java": "java",
   "kafka": "Kafka",
   "kafka_instance": "$t(common:kafka) Instances",
   "kafka_instance_name": "$t(common:kafka) instance name",
+  "loading": "Loading",
+  "mvn": "mvn",
+  "name": "Name",
+  "no_permission_to_connect_registry": "Ask the Service Registry instance owner for connection information.",
+  "no_permission_to_delete_service_registry": "Organization administrators and the Service Registry instance owner can delete this instance.",
+  "no_results_found": "No results found",
+  "options_toggle": "Items per page",
   "owner": "Owner",
+  "page": "page",
+  "per_page_suffix": "per page",
+  "preparing": "Creation in progress",
   "progress_stepper_current_step": "{{currentStep}} of {{total}} steps completed",
+  "provisioning": "Creation in progress",
+  "quick_start_guide_message": "Need help getting started? Follow our quick start guide.",
+  "ready": "Ready",
   "refresh_button_label": "Refresh",
   "region": "Region",
+  "registry_instance_list": "Service Registry instance list",
+  "request_rate": "Request rate",
+  "request_rate_value": "up to 10 requests/second",
   "required": "Required",
+  "reset_credentials": "Reset credentials",
+  "resources": "Resources",
   "return_to_home_page": "Return to home page",
+  "sample_code": "Sample code",
   "search_button_label": "Search",
   "select": "Select",
   "service_registry": "Service Registry",
+  "service_registry_creation_failed_alert_title": "Service Registry instance can't be created",
+  "service_registry_name_label": "Type <b>{{name}}</b> to confirm:",
   "servicePreview": "Service preview ",
   "servicePreviewTooltip": "Service preview releases provide early access to a limited set of features that might not be fully tested. Users should not use service preview software in production or for business-critical workloads.",
   "size": "Size",
   "skeleton_loader_screenreader_text": "Loading {{contentName, content}}",
   "something_went_wrong": "Something went wrong",
+  "standard_service_registry_alert_message": "Your organization reached the limit of Service Registry instances available for creation.",
+  "standard_service_registry_alert_title": "Service Registry instances are unavailable for creation",
+  "standard_service_registry_creation_failed_alert_message": "Your organization reached the limit of Service Registry instances available for creation.",
```

Comment:
if this alert are specify to service registry, I think we can move this to service-registry.json file